### DISCUSSION TITLE
elastalert  1 matches, 1 alerts sent but alert is not getting displayed the web interface

### DISCUSSION THREAD

**Discussion by Fad-Hiba** [Security-Onion-Solutions/securityonion#13462] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Could you Please help me !                                                                                                                                                                      I created a sigma rule, but I can't see the result in the alert interface. Logs in elastalert.log provided below indicate that the rule is  functioning and matching events and elastalert sent them to be displayed, but no result is shown in ALERT

---------------------------------------------------------------------------------------------------------------------------------------------
2024-08-11 12:33:00,923     INFO apscheduler.executors.default Running job "Rule: WindowsSecurityEventCleared - 137e31cbf (trigger: interval[0:03:00], next run at: 2024-08-11 12:36:04 UTC)" (scheduled at 2024-08-11 12:33:00.922635+00:00)
2024-08-11 12:33:00,956     INFO           elastalert Queried rule WindowsSecurityEventCleared - 137e31cbf from 2024-08-11 12:23 UTC to 2024-08-11 12:33 UTC: 3 / 3 hits
2024-08-11 12:33:01,003     INFO           elastalert Ran WindowsSecurityEventCleared - 137e31cbf from 2024-08-11 12:23 UTC to 2024-08-11 12:33 UTC: 3 query hits (2 already seen), 1 matches, 1 alerts sent
2024-08-11 12:33:01,003     INFO           elastalert WindowsSecurityEventCleared - 137e31cbf range 600
----------------------------------------------------------------------------------------------------------------------------------------------

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
You post indicated that you're running 2.4.50 -- have you considered updating to the current version? Starting in 2.4.70, Playbook was replaced by Detections as the platform for handling Sigma rules, so any troubleshooting under this old version would be wasted time when you update.

**Comment by Fad-Hiba** [comment]
I have another instance of securityonion standalone 2.4.90 for tests and I'm having exactly the same problem, have I missed any details in the configuration? Please let me know how to solve it.

**Reply by InfosecGoon** [reply]
If you open the rule in Detections and go to the Detection Logic tab, there's a Convert button at the bottom - click on that, and then the "Test in Kibana" button that comes up. When you run the EQL query in Kibana, does it return results?
